_sidebar.md

```diff
@@ -29,8 +29,7 @@
 * [[Bug Fixes]] 🐛
 * [[Feature Requests]] 📫
 * [[Create and Submit PullRequest]] 🚀
-* [[Add Documentation]] 📝
-* [[Improve Documentation]] ♻️
```

Comment:
I believe there's a couple of pages which reference these as well, so we should do a quick search and make sure to update the links to point to the new page. Thanks!

Reply:
Not able to find any reference to these pages. I checked the docs repo as well.

Reply:
Just saw the one in Create and Submit PR link "Improving Documentation" which was the wrong link anyway I added (oops). Thanks!